apps/backend/core/configuration/validation/orchestrator.py

```
@@ -0,0 +1,467 @@
+            >>> report = await orchestrator.validate_all(parallel=False)
+        """
+        # Default fail_fast to production mode
+        if fail_fast is None:
+            fail_fast = self.config.is_production
+
+        start_time = perf_counter()
+        results: list[ValidationResult] = []
+
+        logger.info(
+            f"Starting validation for {len(self.validators)} services "
+            f"(parallel={parallel}, fail_fast={fail_fast})"
```

Ruff (0.14.4)

Logging statement uses f-string

(G004)